exonum/src/proto/schema/exonum/key_value_sequence.proto

```diff
@@ -26,6 +26,7 @@ message KeyValue {
   bytes value = 2;
 }
 
-message OrderedMap {
+// A sequence of repeated key-value pairs.
```

Comment:
Nit: "repeated" looks redundant; "sequence" implies that the field is repeated, and it's seen from the type declaration.